website/source/docs/providers/ultradns/r/record.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "ultradns"
+page_title: "UltraDNS: ultradns_record"
+sidebar_current: "docs-ultradns-resource-record"
+description: |-
+  Provides a UltraDNS record resource.
+---
+
+# ultradns\_record
+
+Provides a UltraDNS record resource.
+
+## Example Usage
+
+```
+# Add a record to the domain
+resource "ultradns_record" "foobar" {
+	zone = "${var.ultradns_domain}"
+	name = "terraform"
+	rdata = [ "192.168.0.11" ]
+	type = "A"
+	ttl = 3600
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
```

Comment:
Maybe we should link out to the UltraDNS docs for these attributes, especially since `rdata` is pretty UDNS specific.

Something like this page maybe? https://restapi.ultradns.com/v1/docs#post-rrset